bedrock/firefox/templates/firefox/firstrun/firstrun-horizon.html

```diff
@@ -55,8 +55,10 @@ <h2 class="larger">{{ _('Sign in to Firefox and you’re good to go.') }}</h2>
           {% endif %}
         </header>
         <div class="fxaccounts-container">
+          {# Bug 1354710: firstrun pages need to pass funnelcake parameters to FxA #}
+          {% set utm_source = 'firstrun' if not funnelcake_id else 'firstrun_f' + funnelcake_id %}
```

Comment:
[Reply in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1354710#c15) is that doing this for all funnelcakes would be useful.